gcloud/_apitools/http_wrapper.py

```diff
@@ -165,7 +163,7 @@ def ProcessContentRange(content_range):
             start, _, end = byte_range.partition('-')
             return int(end) - int(start) + 1
 
-        if '-content-encoding' in self.info and 'content-range' in self.info:
+        if 'content-encoding' in self.info and 'content-range' in self.info:
```

Comment:
`'-content-encoding'` is not a valid HTTP header:  `apitools` has a typo here.

Re; `bytes` vs. `str`:  the response class follows `httplib2`.